glossary.html

```diff
@@ -1,9 +1,39 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 <head>
-    <title></title>
+    <title>YangZhou</title>
+    <meta charset="utf-8">
 </head>
 <body>
-
+    <header>
+        <nav>
+            <ul>
+                <li><a href="/CSD120/index.html">home</a> </li>
+                <li><a href="/CSD120/links.html">Links</a></li>
+                <li><a href="/CSD120/glossary.html">glossary</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <h1>Acronym Glossary</h1>
+        <dl>
+            <dt>HTML</dt>
+            <dd>hypertext markup language</dd>
+            <dt>HTTP</dt>
+            <dd>Hypertext Transfer Protocol</dd>
+            <dt>IP</dt>
+            <dd>Internet Protocol</dd>
+            <dt>TCP</dt>
+            <dd>Transmission Control Protocol</dd>
+        </dl>
```

Comment:
Note that the definition list has a single `<dl>` element with pairs of `<dt>` and `<dd>` elements nested inside.